src/env/prod/terraform.tfvars

```diff
@@ -327,3 +329,13 @@ reporting_analysis_function_always_on         = true
 reporting_analysis_function_autoscale_minimum = 1
 reporting_analysis_function_autoscale_maximum = 3
 reporting_analysis_function_autoscale_default = 1
+
+# canone unico
+canoneunico_plan_kind     = "Linux"
```

Comment:
Since this value does not change among the environment you could set it as a default in varriable.tf file